database.go

```diff
@@ -1049,6 +1049,35 @@ func (rp *RetentionPolicy) shardGroupByID(shardID uint64) *ShardGroup {
 	return nil
 }
 
+// dropMeasurement will remove a measurement from the index.
+func (db *database) dropMeasurement(name string) error {
```

Comment:
My understanding is the meta store and the shards are different buckets.  Here I need to walk the shards by series that belonged to that measurement to drop them.  In the meta store I can just drop the measurement bucket.  I could of missed something though.